src/main/kotlin/InputIO.kt

```diff
@@ -0,0 +1,33 @@
+import java.lang.NumberFormatException
+
+class InputIO {
+    fun inputNumber(numberType: InputType): Int {
+        inputView(numberType)
+
+        val inputValue = checkNumber(readlnOrNull())
+        if (inputValue != null)
+            return inputValue
+
+        return inputNumber(numberType)
```

Comment:
Elvis 연산자를 이용하면 아래와 같이 표현할 수 있습니다.

```suggestion
        return checkNumber(readlnOrNull())
            ?: return inputNumber(numberType)
```